pkg/sql/sem/builtins/pg_builtins.go

```diff
@@ -1871,6 +1871,8 @@ SELECT description
 				"REFERENCES WITH GRANT OPTION": {privilege.SELECT, true},
 				"TRIGGER":                      {privilege.CREATE, false},
 				"TRIGGER WITH GRANT OPTION":    {privilege.CREATE, true},
+				"RULE":                         {privilege.RULE, false},
+				"RULE WITH GRANT OPTION":       {privilege.RULE, false},
```

Comment:
nit: this should be `{privilege.RULE, true}` (the true is for WITH GRANT OPTION)

although I notice that for other entries like TRIGGER and REFERENCES, we instead alias to another privilege. curious if @otan thinks we should go with that approach for RULE?